MonadicBits/Either.cs

```diff
@@ -94,18 +114,25 @@ namespace Either
         {
             public T Value { get; }
 
-            public Left(T value) => Value = value;
+            private Left(T value) => Value = value;
```

Comment:
backwards compatibility given?

Reply:
IMHO this in not important as we are still in alpha and anything can change at any point. Problems wound be easy to fix. We are still improving on the general API.

Reply:
going to undo this anyways

Reply:
In this PR? Or upcoming?

Reply:
already did